rllib/connectors/env_to_module/frame_stacking.py

```diff
@@ -0,0 +1,116 @@
+from functools import partial
```

Comment:
Built-in framestacking connector to use for Atari.

Optionally move this into `examples` folder as RLlib does not use this automatically (user has to explicitly configure this connector via `config.env_to_module_connector = lambda env: FrameStackingEnvToModule(env=env, num_frames=4)`).